src/NzbDrone.Core/Localization/Core/en.json

```
@@ -1842,6 +1842,14 @@
   "SeasonFinale": "Season Finale",
   "SeasonFolder": "Season Folder",
   "SeasonFolderFormat": "Season Folder Format",
+  "SeasonImportAllowAnyWarning": "Allow a season pack if it upgrades any episode. Note that this may result in stuck queue items.",
+  "SeasonImportAllowHelpText": "Require a season pack to be a quality or custom format upgrade for all episodes",
+  "SeasonImportAllowLabel": "Allow Season Importing",
```

```suggestion
  "SeasonImportAllowLabel": "Allow Season Pack Importing",
```

is this only allowing season pack importing for packs where not all are upgrades, or is this new setting required for any season packs?